package.json

```diff
@@ -37,24 +37,30 @@
   },
   "homepage": "https://github.com/energywebfoundation/ew-asset-registry-lib#readme",
   "dependencies": {
-    "ew-asset-registry-contracts": "0.0.99",
-    "ew-user-registry-contracts": "0.0.99",
-    "ew-user-registry-lib": "0.0.99",
-    "ew-utils-general-lib": "0.0.100",
+    "ew-user-registry-lib": "1.0.2",
+    "ew-utils-general-lib": "1.0.1",
     "ew-utils-testbackend": "v0.0.99",
     "web3": "1.0.0-beta.37",
     "winston": "3.2.1"
   },
   "devDependencies": {
-    "@types/chai": "^4.1.4",
-    "@types/mocha": "^2.2.48",
-    "@types/node": "^9.6.28",
+    "@types/chai": "4.1.4",
+    "@types/mocha": "2.2.48",
+    "@types/node": "9.6.28",
     "@types/web3": "1.0.14",
-    "chai": "^4.1.2",
-    "ganache-cli": "^6.1.8",
-    "mocha": "^5.2.0",
-    "typescript": "^2.9.2",
-    "typescript-json-schema": "^0.31.0"
+    "chai": "4.1.2",
+    "ethlint": "^1.2.4",
+    "ganache-cli": "6.1.8",
+    "lint-staged": "8.1.7",
+    "mocha": "5.2.0",
+    "prettier": "1.17.1",
+    "truffle": "^5.0.18",
```

Comment:
Using not-exact dependency version for Truffle or any other dependency is not a good idea, because it's often the case that project breaks because of dependency update